lua/swamp/swampshop/sh_products.lua

```diff
@@ -7,6 +7,7 @@ function SS_Product(product)
 
     function product:CannotBuy(ply)
         if ply.SQLCreatingItem then return "Database lock, try again." end
+        if self.class == "prop_physics" then ply:Notify('Cannot spawn props while dead.') return end
```

Comment:
uh huh...

Okay so. You're trying to prevent something from being done while dead. In what place in your code do you check `player:Alive()` which would be logically mandatory to have a condition like that?

Reply:
Whoops, I forgot the alive function. Unless there is somewhere else you're suggesting I put this?